### PR TITLE
Disable camera for not HTTPS #306

### PR DESCRIPTION
**Pull request by ZNick1982**
Done with task.
Added check for chrome and for http connection according task

Spent 3 hours

### CODE REVIEW

**Comment by ZNick1982** [comment]
Note: the localization strings added only for English and Russian languages.

File: app/assets/javascripts/uploadcare/locale/en.js.coffee [hunk 1]
```diff
@@ -74,6 +74,9 @@ uploadcare.namespace 'locale.translations', (ns) ->
           notFound:
             title: 'No camera detected'
             text: 'Looks like you have no camera connected to this device.'
+          notHttps:
+            title: 'Camera does not allowed for not protected connections'
```
Comment (dmitry-mukhin) [inline app/assets/javascripts/uploadcare/locale/en.js.coffee]: to be or not to be

**Comment by homm** [comment]
There are should be two separate pull-requests for Camera and intialization errors.

**Comment by ZNick1982** [comment]
Ok, I`ll fix it tonight

**Comment by homm** [comment]
The main Idea was to disable camera source at all. I.e. to hide entire tab, not to show error message to the user. Sorry for not make it clear.

Error message is meaningless for end-user, because he can't do anything with this. Site developer should see the message and decide to exclude the camera tab or to switch to the HTTPS.

**Comment by homm** [comment]
Maybe almost all code should be in `__checkCompatibility` function. Also there should not be any user agent checks. It should be disabled for all browsers.

**Pull request closed without merging**